playwright/tests/e2e/steps/transaction-detail.step.ts

```diff
@@ -0,0 +1,142 @@
+import { createBdd } from "playwright-bdd";
+
+import { blockfrostApi } from "playwright/api/call-blockfrost/blockfrost.api";
+
+import { transactionDetailPage } from "../../../pages/transaction-detail.page";
+
+const { Given, When, Then } = createBdd();
+
+const CONTRACT_TRX_DETAIL_HASH = "591ed189690aa79cc20e66e388b6620d5336cb076577da8896365bb35868549a";
```

Comment:
We need to avoid use hardcoded values, cause in this way the test always are going to be executed with the same test data and this doesn't give us confidence to find possible new issues.

In the Tests document you can find a column call **preconditions** this can give you an idea in how to get specific transactions. For example:

For a smart contract transaction you can get the transaction from the Native scripts and smart contracts section, so first navigate to this section, open a smart contract and then select on of the transactions there. In this same way we can do it for the other transactions just by navigating to other sections of the explorar portal where we know this data can be found.